stdlib/mexpr/pprint.mc

```diff
@@ -148,7 +148,24 @@ let _record2tuple = lam tm.
 -- TERMS --
 -----------
 
-lang PrettyPrint
+lang IdentifierPrettyPrint
+  sem pprintConString =
+  sem pprintVarString =
+  sem pprintLabelString =
+end
+
+lang MExprIdentifierPrettyPrint = IdentifierPrettyPrint
+  sem pprintConString =
+  | s -> _pprintConString s
```

Comment:
I think there are now two things changing how identifiers are displayed; the functions in this fragment, and the renaming to avoid conflicts. Can the latter be changed, and how does it interact with these functions?

(I'm guessing this might be more of a question for the meeting, when @dlunde is present)

Reply:
We discussed this so we leave it as is for now.